src/app/[locale]/(pages)/accedi/errore/page.tsx

```diff
@@ -110,10 +108,7 @@ const LoginErrorPage = () => {
                   </Button>
                 </Grid>
                 <Grid item xs={6} justifySelf="center">
-                  <Button
-                    onClick={() => router.push(`${ROUTES.LOGIN}`, { locale: localeFromStorage })}
-                    variant="contained"
-                  >
+                  <Button onClick={() => pushWithLocale(`${ROUTES.LOGIN}`)} variant="contained">
```

Comment:
form me need to delete all occurrency into the code  of this `${

Reply:
Removed all